collectors/python.d.plugin/postgres/postgres.conf

```diff
@@ -68,8 +68,15 @@
 #     password            : 'example_pass'
 #     host                : 'localhost'
 #     port                : 5432
-#     connect_timeout     : 2     # in seconds, default is 2
-#     statement_timeout   : 2000  # in ms, default is 2000
+#     connect_timeout     : 2                  # in seconds, default is 2
+#     statement_timeout   : 2000               # in ms, default is 2000
+#
+# SSL connection parameters (https://www.postgresql.org/docs/9.1/libpq-ssl.html)
```

Comment:
9.1 is deprecated, you should use "current" documentation : https://www.postgresql.org/docs/current/libpq-ssl.html

Reply:
i see nothing has been changed since 9.1 for ssl, same modes, same params, so the only ivalid thing is link.

Thank you @anayrat

Reply:
@anayrat btw if there is a chance you could test it would be very nice